uint8 type1 #type of error 
int16 int1
int16 int2
int16 int3
int16 int4
string string1
string string2
string string3
string string4
